Mathlib/Algebra/BigOperators/Finprod.lean

```diff
@@ -287,6 +287,11 @@ theorem MonoidHom.map_finprod_of_injective (g : M →* N) (hg : Injective g) (f
 theorem MulEquiv.map_finprod (g : M ≃* N) (f : α → M) : g (∏ᶠ i, f i) = ∏ᶠ i, g (f i) :=
   g.toMonoidHom.map_finprod_of_injective (EquivLike.injective g) f
 
+@[to_additive]
+theorem MulEquivClass.map_finprod {F : Type*} [EquivLike F M N] [MulEquivClass F M N] (g : F)
```

Comment:
Do you want both `f` and `g` to be explicit? In all call-cases I can imagine, they can be inferred by the goal.